scripts/if.js

```diff
@@ -43,4 +43,22 @@ function getMonth() {
   } else if (userMonth >= 10 && userMonth <= 12) {
     alert(`It is ${seasons[3]}`);
   }
+}
+
+function checkNumber() {
+  const userNumber = prompt('Enter 5 digits', 12345) || '0';
```

Comment:
```javascript
  const userNumber = prompt('Enter 5 digits', '12345')
  ```
  You've already put fallback here

Reply:
Yes, thank you)

Reply:
Or not?

Reply:
Not, there is a difference with || "0" and without || "0".